frontend/packages/pipelines-plugin/src/components/repository/sections/WebhookSection.tsx

```diff
@@ -155,6 +157,20 @@ const WebhookSection: React.FC<WebhoookSectionProps> = ({ pac, formContextField
       <RadioGroupField
         name={`${fieldPrefix}webhook.method`}
         label={t('pipelines-plugin~Secret')}
+        labelIcon={
+          <Tooltip
+            position="right"
+            content={
+              <p>
+                {t(
+                  'pipelines-plugin~The secret is used to attach the webhook to the Git repository.',
```

Comment:
As discussed, the secret is also required to set the Build status on GitHub, GitLab and bitbucket. Without that secret the webhook fails with this error:

```
{"level":"error","ts":"2023-08-10T12:13:06.467Z","logger":"pipelinesascode","caller":"events/emit.go:46","msg":"There was an error while processing the payload: failed to find secret in git_provider section in repository spec: christoph/bitbucket-nodeinfo-pac","provider":"bitbucket-cloud","event-id":"","event-sha":"a902882a9ff9bd1cfc8b36d9ac538d9c9bdbcaba","event-type":"","namespace":"christoph","stacktrace":"github.com/openshift-pipelines/pipelines-as-code/pkg/events.(*EventEmitter).EmitMessage\n\t/go/src/github.com/openshift-pipelines/pipelines-as-code/pkg/events/emit.go:46\ngithub.com/openshift-pipelines/pipelines-as-code/pkg/pipelineascode.(*PacRun).Run\n\t/go/src/github.com/openshift-pipelines/pipelines-as-code/pkg/pipelineascode/pipelineascode.go:56\ngithub.com/openshift-pipelines/pipelines-as-code/pkg/adapter.(*sinker).processEvent\n\t/go/src/github.com/openshift-pipelines/pipelines-as-code/pkg/adapter/sinker.go:52\ngithub.com/openshift-pipelines/pipelines-as-code/pkg/adapter.listener.handleEvent.fu...
{"level":"error","ts":"2023-08-10T12:13:06.478Z","logger":"pipelinesascode","caller":"events/emit.go:46","msg":"Cannot create status: failed to find secret in git_provider section in repository spec: christoph/bitbucket-nodeinfo-pac: no token has been set, cannot set status","provider":"bitbucket-cloud","event-id":"","event-sha":"a902882a9ff9bd1cfc8b36d9ac538d9c9bdbcaba","event-type":"","namespace":"christoph","stacktrace":"github.com/openshift-pipelines/pipelines-as-code/pkg/events.(*EventEmitter).EmitMessage\n\t/go/src/github.com/openshift-pipelines/pipelines-as-code/pkg/events/emit.go:46\ngithub.com/openshift-pipelines/pipelines-as-code/pkg/pipelineascode.(*PacRun).Run\n\t/go/src/github.com/openshift-pipelines/pipelines-as-code/pkg/pipelineascode/pipelineascode.go:58\ngithub.com/openshift-pipelines/pipelines-as-code/pkg/adapter.(*sinker).processEvent\n\t/go/src/github.com/openshift-pipelines/pipelines-as-code/pkg/adapter/sinker.go:52\ngithub.com/openshift-pipelines/pipelines-as-code/pkg/adapter.listener...
```

```suggestion
                  'pipelines-plugin~The secret is required to set the build status and to attach the webhook to the Git repository.',
```